Makefile

```diff
@@ -9,14 +9,33 @@ audit:
 	go list -m all | nancy sleuth
 
 .PHONY: build
-build:
+build: generate-prod
 	go build -tags 'production' -o $(BINPATH)/dp-frontend-cookie-controller -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
 
 .PHONY: debug
-debug:
+debug: generate-debug
 	go build -tags 'debug' -o $(BINPATH)/dp-frontend-cookie-controller -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
 	HUMAN_LOG=1 DEBUG=1 $(BINPATH)/dp-frontend-cookie-controller
 
 .PHONY: test
-test:
+test: generate-debug
 	go test -race -cover ./...
+
+.PHONY: generate-debug
+generate-debug: fetch-renderer-lib
+	# fetch the renderer library and build the dev version
+	cd assets; go run github.com/kevinburke/go-bindata/go-bindata -prefix $(CORE_ASSETS_PATH)/assets -debug -o data.go -pkg assets locales/... templates/... $(CORE_ASSETS_PATH)/assets/locales/... $(CORE_ASSETS_PATH)/assets/templates/...
+	{ echo "// +build debug\n"; cat assets/data.go; } > assets/debug.go.new
+	mv assets/debug.go.new assets/data.go
+
+.PHONY: generate-prod
+generate-prod: fetch-renderer-lib
+	# fetch the renderer library and build the prod version
+	cd assets; go run github.com/kevinburke/go-bindata/go-bindata -prefix $(CORE_ASSETS_PATH)/assets -debug -o data.go -pkg assets locales/... templates/... $(CORE_ASSETS_PATH)/assets/locales/... $(CORE_ASSETS_PATH)/assets/templates/...
+	{ echo "// +build production\n"; cat assets/data.go; } > assets/data.go.new
+	mv assets/data.go.new assets/data.go
+
+.PHONY: fetch-renderer-lib
+fetch-renderer-lib:
```

Comment:
As I see it, we need to fetch the renderer library first in order to build the assets data file (if there's a tidier way to do write this up in the Makefile, I'm all keen to know)

We run `go-bindata` with the `CORE_ASSETS_PATH` included, in order to build both app-specific and core assets into a single `data.go` output.

The `-prefix` flag removes `${CORE_ASSETS_PATH}/assets` from the core assets that are pulled in from the library. This means that the core asset names in data.go can be referenced in our code without requiring the full path - (i.e. `main.tmpl` instead of `/path/to/main.tmpl`)